jstests/core/explain_use_backup_plan.js

```diff
@@ -0,0 +1,28 @@
+// Test that the explain will use backup plan if the original winning plan ran out of memory in the
+// "executionStats" mode
+// This test was designed to reproduce SERVER-32721"
+(function() {
+    "use strict";
+
+    db.foo.drop() let bulk = db.foo.initializeUnorderedBulkOp();
+    for (let i = 0; i < 100000; ++i) {
+        bulk.insert({_id: i, x: i, y: i});
+    }
+
+    bulk.execute();
+    db.foo
+        .ensureIndex({x: 1})
+
+        // Configure log level and lower the sort bytes limit.
+        db.setLogLevel(5, "query");
+    db.adminCommand({setParameter: 1, internalQueryExecMaxBlockingSortBytes: 100});
+
+    // This query will not use the backup plan, hence it generates only two stages: winningPlan and
+    // rejectedPlans
+    assert
+        .commandWorked(db.foo.find({x: {$gte: 90}}).sort({_id: 1}).explain(true))
```

Comment:
Your comment here mentions what we expect, but you just assert that the command "worked", which just means it returned a document with an `ok` field with a truthy value (usually `1`). 

Check out the helper functions defined here: https://github.com/mongodb/mongo/blob/r3.7.2/jstests/libs/analyze_plan.js

I think you should add a helper there which gets the 'originalWinningPlan', and assert that it is null here, and non-null below.